### PR TITLE
Add Users To Groups using Server-Side Scripts

### PR DESCRIPTION
**Pull request by prashantmrshine**

### CODE REVIEW

File: GlideRecord/Add n number of users to n number of groups using server scripts/AddUserstoGroups.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+        var grMember = new GlideRecord('sys_user_grmember'); // Query for Group Membership table
+        grMember.initialize();
+        grMember.group = groupSysId;
+        grMember.user = userSysId;
+        grMember.insert(); // Users will be inserted into the Group Membership table
+
+        gs.info('Added User ' + userSysId + 'to group ' + groupSysId); // Info Message displaying after each users group membership creation
+    }
+
```
Comment (msdfib) [inline GlideRecord/Add n number of users to n number of groups using server scripts/AddUserstoGroups.js:19]: Missing bracket

**Comment by msdfib** [comment]
A similar script is already in repository #1300, although I like your version better.

File: GlideRecord/Add n number of users to n number of groups using server scripts/AddUserstoGroups.js (hunk 1 above)
Comment (prashantmrshine) [reply GlideRecord/Add n number of users to n number of groups using server scripts/AddUserstoGroups.js]: I have updated the bracket on my script

**Review by SapphicFire — APPROVED** [review]
Similar to other submissions but improves and works with multiple users and groups.

Looks good to me, thank you

**Pull request merged**